elevate-cpanel

```diff
@@ -871,14 +904,26 @@ sub run_stage_3 ($self) {
 
             $self->run_once(
                 install_pkgs => sub {
-                    unless ( Cpanel::Pkgr::is_installed('elevate-release') ) {
+                    my $leap_data_pkg = 'leapp-data-almalinux';
+                    if ($self->upgrade_to_rocky()) {
+                        $leap_data_pkg = 'leapp-data-rocky';
+                    }
+                    elsif ($self->upgrade_to_cloudlinux()) {
+                        $leap_data_pkg = 'leapp-data-cloudlinux';
+                    }
 
+                    if ($self->upgrade_to_cloudlinux()) {
+                        # CloudLinux upgrade currently only functions with elevate-testing
+                        ssystem_and_die(qw{/usr/bin/curl https://repo.almalinux.org/elevate/testing/elevate-testing.repo -o /etc/yum.repos.d/elevate-testing.repo});
```

Comment:
Is this a temporary workaround? I'm not wild about this going in prior to you releasing it from the testing repository.

Reply:
Makes sense. If we wanted to release things into publicly available beta-testing, maybe we should fork `elevate-cpanel` as well, and provide it to users as a testing release from our fork?